docs/java/guides/logging-overview.md

```diff
@@ -3,36 +3,88 @@ id: logging-overview
+- How the SDK integrates with logging frameworks
+- How to create log entries
+- How to configure those entries
 
 To better differentiate between different kinds of logging we will call the type of logs that are usually only relevant for developers to understand why the system behaves the way it did (for example during debugging) as _Application Logging_.
 Other types of logs might be kept due to legislative requirements (audit trails or audit logging) or be part of the terms and conditions (business logging).
+This document describes how to write and configure _Applications Logs_.
 
-[//]: # (Add some points of the history of logging in Java)
+## Logging Overview
 
-[//]: # (Add a section on what SLF4J is)
+For creating log entries the SDK relies on the popular [Simple Logging Facade for Java (SLF4J)](http://www.slf4j.org/).
+It serves as an interface to a variety of different logging frameworks (e.g. [Logback](http://logback.qos.ch/), [log4j](https://logging.apache.org/log4j/2.x/)).
 
-### Write log messages in your application
+That means there are two components involved:
 
-To start writing log messages you need two dependencies:
+- The _SLF4J API_
+  
+  The API is used to get a logger instance and create log entries:
 
-- The SLF4J API `org.slf4j:slf4j-api`
-- A logging implementation of your choice (e.g. logback `ch.qos.logback:logback-classic`)
+  ```java
+  Logger logger = LoggerFactory.getLogger(DummyClass.class);
+  logger.debug("message");
+  ```
 
-Depending on the chosen logging implementation you might not _need_ to specify the SLF4J API, but it's in general best practice to not rely on transitive dependencies and therefore reference the SLF4J API anyway.
+- A _logging framework_ which provides the implementation of that API
+  
+  The framework is then responsible for writing such messages according to a configuration.
+  Which configuration options are available depends on the framework.
+  Typically one can configure a log level (Error, Warn, Debug, etc.) and the format of messages.
+
+The SDK itself _only_ relies upon the SLF4J API, not on any specific logging framework.
+This is good practice because otherwise all consumers would be forced to use the same logging framework that the SDK comes with, rendering the SDK unusable for many use cases.
+
+That means that you have to _provide a logging framework_ in your application.
+Otherwise you will not see any log entries the SDK attempts to put out.
+
+:::tip
+In case your project is based on one of the SDK archetypes you will already have Logback set up as the logging provider.
+:::
+
+### Providing a Logging Framework
+
+In order to provide a logging framework you have to add it to the dependency tree.
+Which artifacts are to be added exactly depends on the framework.
+
+To take Logback as an example:
+
+```xml
+<dependency>
+	<groupId>ch.qos.logback</groupId>
+	<artifactId>logback-classic</artifactId>
+	<version>latest-logback-verison</version>
```

Comment:
Small typo
```suggestion
	<version>latest-logback-version</version>
```

Reply:
Also, scope `runtime` should probably be used here.
```suggestion
	<version>latest-logback-verison</version>
	<scope>runtime</scope>
```

Reply:
Yes, I was a bit unsure here. If the scope is `runtime`, would it be available when running tests?

Reply:
Yep, it is: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope